<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QMainWindow" name="PreferencesDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>150</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="generalPage">
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::FieldsStayAtSizeHint</enum>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="fileLaunchActionLabel">
          <property name="text">
           <string>File Launch Action</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="fileLaunchActionComboBox">
          <item>
           <property name="text">
            <string>Do nothing</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Open last file</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Show open file dialog</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="fileHandlingLabel">
          <property name="text">
           <string>File Handling</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="autoReloadCheckBox">
          <property name="toolTip">
           <string>When checked, any file currently loaded will be automatically refreshed if it is changed by an external application. This allows you to see your changes immediately simply by pressing save in your text editor.</string>
          </property>
          <property name="text">
           <string>Automatically reload changed files</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="debugPage">
       <layout class="QGridLayout" name="gridLayout">
        <property name="verticalSpacing">
         <number>4</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelLoggers">
          <property name="text">
           <string>Loggers:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelChannels">
          <property name="text">
           <string>Channels:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QListWidget" name="listWidgetLoggers">
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QListWidget" name="listWidgetLoggerChannels">
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollPerPixel</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="generalAction"/>
   <addaction name="debugAction"/>
  </widget>
  <action name="generalAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/General.png</normaloff>:/icons/General.png</iconset>
   </property>
   <property name="text">
    <string>General</string>
   </property>
  </action>
  <action name="debugAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/icons/Debug.png</normaloff>:/icons/Debug.png</iconset>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>fileLaunchActionComboBox</tabstop>
  <tabstop>autoReloadCheckBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileLaunchActionComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>preferenceChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>generalAction</sender>
   <signal>triggered()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>switchPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>debugAction</sender>
   <signal>triggered()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>switchPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoReloadCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>preferenceChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidgetLoggers</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>selectedLoggerChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidgetLoggers</sender>
   <signal>itemChanged(QListWidgetItem*)</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>loggerActivationChanged(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>accept()</slot>
  <slot>reject()</slot>
  <slot>preferenceChanged()</slot>
  <slot>switchPage()</slot>
  <slot>loggerActivationChanged(QListWidgetItem*)</slot>
  <slot>selectedLoggerChanged()</slot>
 </slots>
</ui>
